warehouse/utils/utils.go

```diff
@@ -784,6 +789,20 @@ func NewCounterStat(name string, extraTags ...Tag) stats.RudderStats {
 	return stats.NewTaggedStat(name, stats.CountType, tags)
 }
 
+func WHCounterStat(name string, warehouse *WarehouseT, extraTags ...Tag) stats.RudderStats {
+	tags := map[string]string{
+		"module":      WAREHOUSE,
+		"destType":    warehouse.Type,
+		"destID":      warehouse.Destination.ID,
+		"whNamespace": warehouse.Namespace,
```

Comment:
Can we cross verify whNamespace because in the alerts we are using namespace.
https://github.com/rudderlabs/rudderstack-operator/blob/4581315f2e65c71747ee8d868dc92e48e27c46b1/helm-charts/rudderstack-monitor/charts/kapacitor/alerts/warehouse-missing-datatype.yaml#L14

Reply:
This `namespace` is kubernetes namespace. Added by operator by default.